metabase/metabase
# waterfall total

Closes https://github.com/metabase/metabase/issues/35812

### Description

Implements the "show total" setting for the waterfall chart. Note that there are two known regressions (https://github.com/metabase/metabase/issues/37055, https://github.com/metabase/metabase/issues/37056) in this PR. In the interest of time I'll leave them open for now, and we can fix them later before finishing the project.

### How to verify

1. Create a waterfall chart and leave the show total setting enabled
2. Add to a dashboard
3. Send a subscription
4. Confirms it renders correctly (total bar should be visible and should display the correct value) 


### Demo

![Screenshot 2023-12-21 at 11.13.07 AM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/9ULpbAzGBtvQRyZEpGvu/b52eade4-bfcd-4d29-b9cd-11494c24d955.png)

### Checklist
